splink/analyse_blocking.py

```diff
@@ -139,3 +139,103 @@ def cumulative_comparisons_generated_by_blocking_rules(
     linker._analyse_blocking_mode = False
 
     return br_comparisons
+
+
+def count_comparisons_from_blocking_rule_pre_filter_conditions_sqls(
+    linker: "Linker", blocking_rule: Union[str, "BlockingRule"]
+):
+    if isinstance(blocking_rule, str):
```

Comment:
FYI, you'll be able to use [this new conversion function](https://github.com/moj-analytical-services/splink/blob/33fe61f646f8c2ec959ad0f393969bcdc46744c7/splink/blocking.py#L16) once some of my BR work has been merged in.